From 4c7a211046a3956058b512edd8533f7df031cc4a Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Thu, 19 Oct 2017 23:49:19 +0200
Subject: [PATCH 040/105] Kconfig: add CONFIG_BROKEN

Provide a Kconfig option that we can use as dependency for
features that are broken. This allows to easily disable them
without removing the code.

As no short text is supplied the option will not appear in
menuconfig.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
Reviewed-by: Simon Glass <sjg@chromium.org>
---
 Kconfig | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/Kconfig b/Kconfig
index 227fb17dd9..e57fad4592 100644
--- a/Kconfig
+++ b/Kconfig
@@ -14,6 +14,12 @@ source "arch/Kconfig"
 
 menu "General setup"
 
+config BROKEN
+	bool
+	help
+	  This option cannot be enabled. It is used as dependency
+	  for broken and incomplete features.
+
 config LOCALVERSION
 	string "Local version - append to U-Boot release"
 	help
-- 
2.13.6

